lib/posix/sleep.c

```diff
@@ -5,14 +5,17 @@
  */
 
 #include <kernel.h>
+#include <limits.h>
+#include <errno.h>
+#include <posix/time.h>
 #include <posix/unistd.h>
 
 /**
  * @brief Sleep for a specified number of seconds.
  *
  * See IEEE 1003.1
  */
-unsigned sleep(unsigned int seconds)
+unsigned int sleep(unsigned int seconds)
```

Comment:
It's good zen exercise to try to avoid patching unrelated (== not directly related) code ;-).

Reply:
Will fix. That was a checkpatch.pl / fix-inline modification.